ontology/yaml/resources/HVAC/entity_types/CHWS.yaml

```diff
@@ -214,6 +214,25 @@ CHWS_SWTC_WDPC2X_BYPVPM_WFRM_CLPM:
   - WFRM
   - CLPM
 
+CHWS_RWTC_CHWVM:
+  id: "16842546300862857217"
```

Comment:
Please remove ID. New entity types do not require IDs (our system sets the IDs automatically).